driver/cosineembeddingloss_driver.hpp

```diff
@@ -547,26 +563,12 @@ int CosineEmbeddingLossDriver<Tgpu, Tref>::RunBackwardCPU()
     return miopenStatusSuccess;
 }
 
-template <typename Tgpu, typename Tref>
-Tref CosineEmbeddingLossDriver<Tgpu, Tref>::GetTolerance()
-{
-    // Computation error of fp16 is ~2^13 (=8192) bigger than
-    // the one of fp32 because mantissa is shorter by 13 bits.
-    auto tolerance = std::is_same<Tgpu, float>::value ? 1.5e-6 : 8.2e-3;
-
-    // bf16 mantissa has 7 bits, by 3 bits shorter than fp16.
-    if(std::is_same<Tgpu, bfloat16>::value)
-        tolerance *= 8.0;
-    return tolerance;
-}
-
 template <typename Tgpu, typename Tref>
 int CosineEmbeddingLossDriver<Tgpu, Tref>::VerifyForward()
 {
     RunForwardCPU();
-    const Tref tolerance = GetTolerance();
-    auto error           = miopen::rms_range(out_host, out);
-
+    auto tolerance = std::numeric_limits<Tgpu>::epsilon() * 10;
```

Comment:
Is there a reason you changed the way tolerance is calculated?

Reply:
I fixed it to ensure the driver has the same tolerance as gtest.